index.html

```diff
@@ -11,26 +10,28 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
-    <title>Portfolio</title>
+    <link rel="shortcut icon" href="images/logo.png" />
+    <title>Rehema's Portfolio</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   </head>
   <body>
     <div class="page-content">
       <header class="header">
         <nav class="nav">
           <div class="mobile-menu">
             <button type="button">
-              <img class="close-image" src="images/x.png" alt="" />
+              <img class="close-image" src="images/x.png" alt="close-icon" />
             </button>
             <div class="mobile-items">
               <a href="#portfolio">Portfolio</a>
               <a href="#about">About</a>
               <a href="#contact">Contact</a>
             </div>
```

Comment:
- [ ] [OPTIONAL] please it would be better if you add the blur to your mobile menu in order to match the Design